### PR TITLE
Ability to import remote files 

### PR DESCRIPTION
**Pull request by cbaykam**
Added the ability to import remote files. 
Fixed assertions with assert_equals to assert_nil 
Fixed a signature error in a file. 
Made tests pass :)

### CODE REVIEW

File: test/test_readers_xls_reader.rb [hunk 1]
```diff
@@ -45,9 +45,10 @@ def test_xls_excel_standardized_open
     def test_txt_in_xls_open
       w = Workbook::Book.new
       w.import(File.join(File.dirname(__FILE__), "artifacts/txt_in_xls.xls"))
-      assert_equal([:naam,:nummer,:ilt,:corporate_key,:naam_medewerker, nil, nil, :telefoon, :openingsdatum],w.sheet.table.header.to_symbols)
-      assert_equal(["dddd",2222,"i9000","asd","Anita",nil,"Betera","012-3456789",Date.new(2011,10,5)],w.sheet.table[1].collect{|a| a.value})
+      assert_equal([:naam, :nummer, :ilt, :corporate_key, :naam_medewerker, nil, nil, :telefoon, :openingsdatum], w.sheet.table.header.to_symbols)
+      assert_equal(["dddd", 2222, "i9000", "asd", "Anita", nil, "Betera", "012-3456789", '5-10-2011'], w.sheet.table[1].collect{|a| a.value})
```
Comment (murb) [inline test/test_readers_xls_reader.rb]: Have to check this one:
a) there used to be a time that specifically formatted csv files got an xls extension just to make them import in excel easily (see e.g. http://www.ebrueggeman.com/blog/generating-php-comma-csv-and-tab-delimited-files )
b) 5-10-2011 is a popular non-US date notation; but Workbook needs a better way of dealing with date parsing

File: test/test_modules_type_parser.rb [hunk 2]
```diff
@@ -16,8 +17,6 @@ def examples
         "n"=>"n",
         "12 bomen"=>"12 bomen",
         "12 bomenasdfasdfsadf"=>"12 bomenasdfasdfsadf",
-        ""=>nil,
-        " "=>nil,
```
Comment (cbaykam) [inline test/test_modules_type_parser.rb]: I think leaving them as is might be better for some csv files. But lib your decision :)

**Comment by murb** [comment]
Hi @cbaykam ! Thanks for the heads up, lots of improvements to the (test)code 👍  I like your change from open to import, which is an improvement (although breaking change, but Workbook is still < 1.0 ;) ). If you can address the issues I mentioned I'll merge it with the existing code base and release a new version of the gem. Thanks again!

File: test/test_modules_type_parser.rb (hunk 2 above)
Comment (murb) [inline test/test_modules_type_parser.rb]: Maybe it should be an explicit option, agree with the deletion :)

Comment (murb) [inline test/test_modules_type_parser.rb]: Ah, now I remember: sometimes excel files are full of crap ('empty'-cells) until the end of the file. I trim the files to make them convenient to handle.

Comment (murb) [inline test/test_modules_type_parser.rb]: Created a new issue

Comment (cbaykam) [inline test/test_modules_type_parser.rb]: I see will change them and submit again.

**Comment by cbaykam** [comment]
Sure will do. And as I get my hands dirty I will clean up the issues reported by hound too :)

**Comment by murb** [comment]
Just don't overdo it ;) Thanks!

**Comment by cbaykam** [comment]
It should be ready.

**Pull request merged**